### PR TITLE
Add last modified date to settings tab

### PR DESCRIPTION
**Pull request by martinlagler**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | https://github.com/sulu/sulu/pull/7238
| License | MIT

#### What's in this PR?

Add new updated date field to the settings tab, with a toggler to enable and disable it.

#### Why?

So articles can have a editable update date.

### CODE REVIEW

File: Content/ArticleResourceItem.php [hunk 1]
```diff
@@ -110,6 +110,14 @@ public function getPublished(): \DateTime
         return $this->article->getPublished();
     }
 
+    /**
```
Comment (martinlagler) [inline Content/ArticleResourceItem.php:113]: I'm not quite sure if this is really needed here.

Comment (wachterjohannes) [reply Content/ArticleResourceItem.php:113]: i think it is not bad to have it!

**Review by wachterjohannes — APPROVED** [review]
As it is BC break - we leave this PR open

**Comment by martinlagler** [comment]
@wachterjohannes I added the lastModified to the sitemap in this PR as well (https://github.com/sulu/SuluArticleBundle/pull/655/commits/f0e99208c50166756ae7de78b72a2c4fd958fa32)

And i noticed that currently the date is not working on the sitemap, because the wrong paramter was set. I will also create a PR for 2.5

File: composer.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
         "handcraftedinthealps/elasticsearch-dsl": "^5.0.7.1 || ^6.2.0.1 || ^7.2.0.1",
         "jms/serializer": "^3.3",
         "jms/serializer-bundle": "^3.3 || ^4.0",
-        "sulu/sulu": "~2.4.13 || ^2.5.9@dev",
+        "sulu/sulu": "2.6.x-dev",
```
Comment (alexander-schranz) [inline composer.json]: We should keep it copmatible to 2.5 atleast. I can have a look at it. Do you know whats breaking there @martinlagler ?

```suggestion
        "sulu/sulu": "~2.4.13 || ^2.5.9@dev",
```

**Comment by alexander-schranz** [comment]
I added the bc layer so it still works for Sulu 2.4, 2.5 and 2.6 has this new feature then enabled. 

See changes in: https://github.com/sulu/SuluArticleBundle/compare/4230b775c73d50afbead9ae6e38130e99eaa53df...fe5c43360d786156ff3828ec4ccae6659fec56c7

**Review by alexander-schranz — APPROVED** [review]

**Review by Prokyonn — APPROVED** [review]

**Pull request merged**